reporting/report.go

```diff
@@ -144,53 +145,60 @@ func writeDB(db string, wd string, showNetNames bool, res *resources.Resources)
 	}
 	res.DB.SelectDB(db)
 
-	err = writeDBHomePage(db)
+	dbMetaInfo, err := res.MetaDB.GetDBMetaInfo(db)
+	if err != nil {
+		return err
+	}
+	maxTime := time.Unix(dbMetaInfo.TsRange.Max, 0).String()
```

Comment:
I messed up in my comment in issue https://github.com/activecm/rita/issues/594#issuecomment-744650240. @Nyhmesis was asking for a timestamp of when the report was generated. I suggested pulling from the database, which is the timestamp contained in the imported log entries. These are both valuable but very different.

For instance, I just generated a report with some sample data from a few years ago. Even though the report was generated today 2021-08-03, it shows a timestamp from 2018, which is when the sample data was captured.

![image](https://user-images.githubusercontent.com/1696711/128093361-0f0d2e43-0871-418f-93d1-632df4e01f18.png)

To make this true to the original issue, could you change it to use `time.Now()` instead? I think this should work (but haven't tested it).

```go
maxTime := time.Now().String()
```

Reply:
Ah, that makes more sense. I'll make those changes.